components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabViewHolder.kt

```diff
@@ -23,7 +21,7 @@ class TabViewHolder(
     itemView: View,
     private val tabsTray: BrowserTabsTray
 ) : RecyclerView.ViewHolder(itemView) {
-    private val cardView: CardView = (itemView as CardView).apply {
```

Comment:
Since we're removing the `CardView`, we should change the type in the xml `mozac_browser_tabstray_item` to not have a `CardView` either, you should use a `FrameLayout`.